apis/python/src/tiledbsoma/dense_nd_array.py

```diff
@@ -20,25 +19,6 @@ class DenseNDArray(NDArray, somacore.DenseNDArray):
     [lifecycle: experimental]
     """
 
-    @property
-    def shape(self) -> NTuple:
```

Comment:
whaaaat? at a glance it looks like these went missing ... i'm trying to see where they went to on this PR ... 👀

Reply:
These were exact duplicates of the methods they overrode; thus they do not need separate documentation after being removed.